sw-sysemu/system.h

```diff
@@ -18,6 +18,9 @@
 #include "esrs.h"
 #include "processor.h"
 #include "testLog.h"
+#if EMU_ERBIUM
+#include "memory/erbium/erbium_reg_region.h"
```

Comment:
I think we have to maintain a bit the architecture of existing sysemu. If you see something similar, e.g. esrs, the functionality is implemented in the root, and the memory side of things is implemented in the memory/ part, that essentially is a collection of all devices, and memory/<device>/main_memory defines how these devices are used.

I haven't time to look at it but this include sounds strange to me.